packages/blade-mcp/knowledgebase/components/Table.md

```diff
@@ -739,3 +739,145 @@ const ServerPaginatedTable = () => {
 
 export default ServerPaginatedTable;
 ```
+
+### Table Nesting Pattern
+
+Hierarchical data display with expandable rows and animations. Use for parent-child relationships or detailed information.
+
+```tsx
+const TableNestingExample = () => {
+  const [expandedRows, setExpandedRows] = useState<Set<string>>(new Set());
+
+  const toggleRow = (id: string) => {
+    const newExpanded = new Set(expandedRows);
+    newExpanded.has(id) ? newExpanded.delete(id) : newExpanded.add(id);
+    setExpandedRows(newExpanded);
+  };
+
+  return (
+    <Table data={tableData}>
+      {(tableData) => (
+        <>
+          <TableHeader>
+            <TableHeaderRow>
+              <TableHeaderCell>Name</TableHeaderCell>
+              <TableHeaderCell>Amount</TableHeaderCell>
+              <TableHeaderCell>Status</TableHeaderCell>
+            </TableHeaderRow>
+          </TableHeader>
+
+          <TableBody>
+            {tableData.map((item) => (
+              <>
+                <TableRow key={item.id} item={item}>
+                  <TableCell>
+                    <Button
+                      variant="tertiary"
+                      size="xsmall"
+                      icon={expandedRows.has(item.id) ? ChevronDownIcon : ChevronRightIcon}
+                      onClick={() => toggleRow(item.id)}
+                    />
+                    {item.name}
+                  </TableCell>
+                  <TableCell>{item.totalAmount}</TableCell>
+                  <TableCell>
+                    <Badge color="positive">{item.status}</Badge>
+                  </TableCell>
+                </TableRow>
+
+                {expandedRows.has(item.id) && (
+                  <TableRow key={`${item.id}-expanded`} item={item}>
+                    <TableCell gridColumnStart={1} gridColumnEnd={4}>
+                      <BaseMotionEntryExit
```

Comment:
Can you check this once? This component is not exported so might create wrong code on consumer side